dot/state/block.go

```diff
@@ -62,6 +62,10 @@ type BlockState struct {
 	unfinalisedBlocks *hashToBlockMap
 	tries             *Tries
 
+	// State variables
+	sync.RWMutex
```

Comment:
Do we need to export this mtx? are we using it outside of `BlockState` ?

Reply:
no we do not, updated